apps/avifenc.c

```diff
@@ -805,6 +792,63 @@ int main(int argc, char * argv[])
         ++argIndex;
     }
 
+    // Check lossy/lossless parameters and set to default if needed.
+    if (lossless) {
+        // Pixel format.
+        if (input.requestedFormat != AVIF_PIXEL_FORMAT_NONE && input.requestedFormat != AVIF_PIXEL_FORMAT_YUV444) {
+            fprintf(stderr,
+                    "When set, the pixel format can only be 444 in lossless "
+                    "mode.\n");
+            returnCode = 1;
+        }
+        input.requestedFormat = AVIF_PIXEL_FORMAT_YUV444; // don't subsample when using
+                                                          // AVIF_MATRIX_COEFFICIENTS_IDENTITY
+        // Quantizers.
+        if (minQuantizer != -1 || maxQuantizer != -1 || minQuantizerAlpha != -1 || maxQuantizerAlpha != -1) {
```

Comment:
We should also allow the quantizers to be set to `AVIF_QUANTIZER_LOSSLESS`.

Note that you allow the other parameters (`input.requestedFormat`, `codecChoice`, and `matrixCoefficients`) to be set to the correct values.

Reply:
Done